index.html

```diff
@@ -19,6 +19,9 @@ <h1>Experts In Construction</h1>
       <section class="services_slider">
         <h2>Latest Services</h2>
         <h3>The Best Services For you.</h3>
+        <h4>Building Contraction</h4>
+        <script defer type="module" src="/slider.js"></script>
```

Comment:
we don't import slider like that here. I will import the script just once, and here will be added a slider's layout

Reply:
so we don't need this line here